lib/coinbase/staking_operation.rb

```diff
@@ -104,6 +110,24 @@ def status
       @model.status
     end
 
+    def terminal_state?
```

Comment:
Probably good to comment the rest of these functions here to be consistent.

Reply:
Aah good catch - refactored it last moment and forgot adding it. Added back - thx!